plugins/modules/openbsd_pkg.py

```diff
@@ -159,6 +160,20 @@ def execute_command(cmd, module):
     return module.run_command(cmd_args, environ_update={'TERM': 'dumb'})
 
 
+def get_all_installed(module):
+    """
+    Get all installed packaged. Used to support diff mode
+    """
+    command = 'pkg_info -Iq'
+
+    rc, stdout, stderr = execute_command(command, module)
+
+    if stderr:
+        module.fail_json(msg="failed in get_all_installed(): " + stderr)
```

Comment:
```suggestion
        module.fail_json(msg="failed in get_all_installed(): %s" % stderr)
```

Reply:
Yeah, I have a much bigger diff where I go through and move to newer styles (f-strings mostly but also some % for more complex things. This was copy-pasta error on my part.

Reply:
Please note that f-strings aren't possible yet for modules since we're still supporting Python 2.7.